internal/federation/server.go

```diff
@@ -481,3 +525,12 @@ func (f *basicKeyFetcher) FetchKeys(
 func (f *basicKeyFetcher) FetcherName() string {
 	return "basicKeyFetcher"
 }
+
+// SupportedRoomVersions is a convenience method which returns a list of the room versions supported by gomatrixserverlib.
+func SupportedRoomVersions() []gomatrixserverlib.RoomVersion {
+	supportedRoomVersions := make([]gomatrixserverlib.RoomVersion, 0, 10)
```

Comment:
Why a capacity of 10 ooc?

Reply:
good question. mostly because I forgot that capacity was optional.

Reply:
... though this change has come from #131 rather than here.